lib/commands/project/project.js

```diff
@@ -39,8 +39,15 @@ var FRAMEWORKS = {
   sails:   { source: 'sails' }
 };
 
-var ASSERTIONTYPES = ['expect', 'should', 'assert'];
-var TESTMODULES = ['supertest', 'request'];
+var TEST_ASSERTION_TYPES = ['expect', 'should', 'assert'];
+var TEST_MODULES = ['supertest', 'request'];
+var TEST_DEPENDENCIES = {
+  'z-schema': {'z-schema': '^3.12.0'},
```

Comment:
Can you remove the nested property? I believe this can be done with a {name: version} hash